pkg/models/datasource_cache.go

```diff
@@ -45,9 +46,16 @@ func (ds *DataSource) GetHttpTransport() (*http.Transport, error) {
 		return t.Transport, nil
 	}
 
+	var tlsSkipVerify, tlsClientAuth, tlsAuthWithCACert bool
+	if ds.JsonData != nil {
+		tlsClientAuth = ds.JsonData.Get("tlsAuth").MustBool(false)
```

Comment:
maybe rename this variable to tlsAuth as well to make the naming more consistent

Reply:
I really wanted to rename `tlsAuth` to `tlsClientAuth`, but there's no good migration path currently.

I kept `tlsClientAuth` in the Go code since it could be changed without breaking existing installations and makes the intent clearer in the code. I'd prefer to keep it as-is to avoid any future regressions.